AgriParcel/schema.json

```diff
@@ -152,6 +152,38 @@
                             "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/EntityIdentifierType"
                         }
                     ]
+                },
+                 "soilTextureType": {
+                    "type": "string",
+                    "description": "Property. Enum: 'Clay', 'Clay, Sandy', 'Clay, Silty', 'Loam', 'Loam, Clay', 'Loam, Sandy', 'Loam, Sandy Clay', 'Loam, Silty', 'Loam, Silty Clay', 'Sand', 'Sand, Loamy', 'Silt'",
+                    "enum": [
+                        "Clay",
+                        "Clay, Sandy",
+                        "Clay, Silty",
+                        "Loam",
+                        "Loam, Clay", 
+                        "Loam, Sandy",
+                        "Loam, Sandy Clay",
+                        "Loam, Silty",
+                        "Loam, Silty Clay",
+                        "Sand",
+                        "Sand, Loamy",
+                        "Silt"
+                    ]
+                },
+                 "irrigationSystemType": {
+                    "type": "string",
+                    "description": "Property. Enum: 'Surface irrigation', 'Localized irrigation', 'Drip irrigation', 'Sprinkler irrigation', 'Center pivot irrigation', 'Lateral move irrigation', 'Sub-irrigation', 'Manual irrigation'",
+                    "enum": [
```

Comment:
Bad idea to include spaces in the types. Unless this is coming from an existing regulation, please remove spaces in the enumeration.